@here/harp-geoutils/lib/projection/EarthConstants.ts

```diff
@@ -10,4 +10,7 @@ export class EarthConstants {
 
     /** The equatorial radius in meters. */
     static EQUATORIAL_RADIUS: number = 6378137.0;
+
+    /** The highest point on earth (Mt. Everest) in meters. */
+    static MAX_HEIGHT: number = 8848.0;
```

Comment:
You also have to map the lowest point(i.e. -433) to 0.0 when creating the texture. If you add the values here, please remove the ones in TerrainElevationRangeSource.ts

Reply:
Done.